test_truststore.py

```diff
@@ -29,6 +30,27 @@
 )
 
 
+@pytest.fixture(scope="session")
+def trustme_ca():
+    if platform.system() == "Windows":
+        pytest.skip("Windows doesn't implement custom CA certificates yet")
+    ca = trustme.CA()
+    yield ca
+
+
+@pytest.fixture(scope="session")
+def httpserver_listen_address():
+    return ("localhost", 8080)
```

Comment:
Minor point, but I tend to end up with too many different projects vying for control of port 8080. It looks like by default pytest-httpserver will listen on localhost on a random available port; why not stick with that and use httpserver.url_for("/") to construct the URL to use for a request?

Reply:
That's a great idea!!